README.md

```diff
@@ -14,7 +14,7 @@
 
 ## What it does
 
-SkyHanni is a Forge mod for Minecraft 1.8.9 that adds many useful features to [Hypixel SkyBlock](https://wiki.hypixel.net/Main_Page). With SkyHanni you have access to:
```

Comment:
you can not just redefine what the mod is about. we are a skyblock mod, have some few generic hypixel features, and thats it.